pulsar-common/src/main/java/org/apache/pulsar/common/topics/TopicList.java

```diff
@@ -78,4 +81,16 @@ public static Set<String> minus(Collection<String> list1, Collection<String> lis
         s1.removeAll(list2);
         return s1;
     }
+
+    private static String removeTopicDomainSchema(String originalRegexp) {
```

Comment:
`removeTopicDomainSchema` -> `removeTopicDomainScheme` or just change to `removeTopicDomain`

Reply:
Renamed